ImageItem/ImageItem.js

```diff
@@ -43,6 +43,16 @@ const ImageItemBase = kind({
 		 */
 		children: PropTypes.node,
 
+		/**
+		 * Sets the position for caption.
+		 * Available positions: 'captionBelow' (default) and 'captionOverlay'.
+		 *
+		 * @type {('captionBelow'|'captionOverlay')}
+		 * @default 'captionBelow'
+		 * @public
+		 */
+		childrenPosition: PropTypes.oneOf(['captionBelow', 'captionOverlay']),
```

Comment:
@adrian-cocoara-lgp 
We're still thinking what is the best name and values for this prop.
We'll let you know after discussed.

Considering the following names
- childrenPosition
- captionBelow & captionOverlay
- captionOverImage

Reply:
Our decision is..
childrenPosition -> captionPosition // `caption` is more clear to app developers
captionBelow -> below // `caption` is already included in the prop name - `captionPosition`
captionOverlay -> overlay // same above
captionOverImage -> captionOverlay // Using just `overlay` could make confused

Reply:
`captionOverImage` was replaced with `childrenPosition`. I don't see any reference to it anywhere, do we still need it?

Reply:
> captionOverImage was replaced with childrenPosition. 

captionOverImage was replaced with captionOverlay
 
> Do you mean the captionOverImage className from less file?

Yes

So the className in computed would be..
```
		className: ({captionPosition, styler}) => styler.append({
			captionOverlay: captionPosition === 'overlay'
		})
```